core-charter.txt

```diff
@@ -1,40 +1,65 @@
+* solutions to facilitate the signalling of transports available to a Device.
 
-The CoRE WG will continue and complete its work on the CoRE Resource Directory (draft-ietf-core-resource-directory), as already partially adopted by OMA LwM2M, and will perform maintenance as well as possible updating, complementing and specification of relevant uses as required. A primary related consideration is the interoperability with DNS-SD and the work of the dnssd WG. The use of CoAP to transport DNS queries and responses will also be investigated. Furthermore, the CoRE WG will also continue and complete its work on enabling broker-based publish-subscribe-style communication over CoAP (draft-ietf-core-coap-pubsub).
+* continue and complete its work on the CoRE Resource Directory 
 
-The CoRE WG will work on related data formats, such as alternative representations of RFC 6690 link format and RFC 7390 group communication information. Also, the CoRE WG will complete its work on Constrained Resource Identifiers for serializing URI components in CBOR (draft-ietf-core-href). Furthermore, the CoRE WG will develop CoRAL (draft-ietf-core-coral), a constrained RESTful application language suitable also for constrained node networks, which comprises an information model and an interaction model, and is intended for driving automated software agents that navigate a Web application. The CoRE WG will complete the Sensor Measurement Lists (SenML) set of specifications, again with consideration to its adoption in OMA LwM2M.
+* consideration of interoperability with DNS-SD and the work of the dnssd WG. 
 
-Besides continuing to examine operational and manageability aspects of the CoAP protocol itself, the CoRE WG will also complete the work on RESTCONF-style management functions available via CoAP that are appropriate for constrained node networks (draft-ietf-core-yang-cbor, draft-ietf-core-sid, draft-ietf-core-comi, draft-ietf-core-yang-library). This requires very close coordination with NETCONF and other operations and management WGs. YANG data models are used for manageability. Note that the YANG modeling language is not a target for change in this process, although additional mechanisms that support YANG modules may be employed in specific cases where significant performance gains are both attainable and required. The CoRE WG will continue to consider the OMA LwM2M management functions as a well-accepted alternative form of management and provide support at the CoAP protocol level where required.
+* use of CoAP to transport DNS queries and responses will also be investigated. 
```

Comment:
I don't think we should do this work in CORE.
I think that DNSOP should do this work.  Of course, CORE will get consulted, but I think that we shouldn't do this.

Reply:
Of course, the two WGs should work together on this.  Doing this right will need to be in CoRE.